XamlControlsGallery/ControlPages/CommandBarPage.xaml

```diff
@@ -42,16 +42,16 @@
             </local:ControlExample.Options>
 
             <local:ControlExample.Xaml>
-                <RichTextBlock Style="{StaticResource XamlCodeRichTextBlockStyle}">
-                    <Paragraph>&lt;CommandBar Background="Transparent" IsOpen="False" DefaultLabelPosition="Right"&gt;</Paragraph>
-                    <Paragraph TextIndent="12">&lt;AppBarButton Icon="Add" Label="Add"/&gt;</Paragraph>
-                    <Paragraph TextIndent="12">&lt;AppBarButton Icon="ReShare" Label="Share"/&gt;</Paragraph>
-                    <Paragraph TextIndent="12">&lt;AppBarButton Icon="Edit" Label="Edit"/&gt;</Paragraph>
-                    <Paragraph TextIndent="12">&lt;CommandBar.SecondaryCommands&gt;</Paragraph>
-                    <Paragraph TextIndent="24">&lt;AppBarButton Icon="Setting" Label="Settings"/&gt;</Paragraph>
-                    <Paragraph TextIndent="12">&lt;/CommandBar.SecondaryCommands&gt;</Paragraph>
-                    <Paragraph>&lt;/CommandBar&gt;</Paragraph>
-                </RichTextBlock>
+                <x:String xml:space="preserve">
+&lt;CommandBar Background="Transparent" IsOpen="False" DefaultLabelPosition="Right"&gt;
```

Comment:
Probably another good opportunity to make the sample dynamically update, but that change shouldn't be part of this PR.

Reply:
https://github.com/Microsoft/Xaml-Controls-Gallery/issues/41